### ISSUE TITLE
When viewing my patterns, I get a servercomm

### ISSUE THREAD

**Issue by heindrichpaul** [gskinner/regexr#295]
I get a servercomm message in the my patterns window instead of my saved patterns. I can at least get my pattern back as I have the share URL

**Comment by wdamien** [comment]
Thanks for letting us know. `servercomm` means theres a generic server error that happened (it should be mapped to something a little more human friendly 😄 )

In this case, it looks like the database is under a heavy load. I'm seeing random requests hit their time limit (currently its set to 5 seconds). A lot of cancelled requests for me still return partial data at <6 seconds.  So we could increase the limit to account for these heavier days. 5 seems to cover the majority of use, but adding a few more seconds can't hurt anything.  So not to worry, your patterns are still there, they're just being a little slow today.

**Comment by dvadneau** [comment]
I haven't been able to save or access patterns consistently for months (well it seemed to not be saving, since I couldn't access them afterwards). I kept getting the servercomm error. I didn't see any mention of this in the GitHub issues and figured it was probably just me/my account or that this wasn't being actively maintained. My apologies for not reporting this much sooner. I don't use this tool on a daily basis - every few weeks/months.

I've been "saving" by bookmarking.

I got the error again today and found this issue.

So I tried refreshing several times and got the servercomm error several times until, eventually, I was able to see my patterns. I then saved some changes and am stuck with the error again.

I do appreciate that this is a tool provided and maintained for free, so again, apologies for not reporting the issue sooner.
